### ISSUE TITLE
redirect Julia's stderr?

### ISSUE THREAD

**Issue by ThomasBreuer** [oscar-system/GAPJulia#95]
Currently Julia error messages are visible on the screen
but they do not appear in GAP's log files.
For example, the `ArgumentError: ...` line from the GAP session
```
gap> Julia.Base.\/\/( 0, 0 );
ArgumentError: invalid rational: zero(Int64)//zero(Int64)
Error, JuliaError
not in any function at *stdin*:15
type 'quit;' to quit to outer loop
brk>
```
is missing from the corresponding GAP log file.

Would it be possible to redirect Julia's stderr
such that GAP's stderr is used instead?

**Comment by sebasguts** [comment]
This was already fixed in #60

**Comment by fingolfin** [comment]
Not really, that only dealt with exceptions caught by us. Eg our test suite causes Julia to print warnings to stderr.
And strictly speaking, I guess one might also want to redirect their stdout

**Comment by sebasguts** [comment]
Okay, the given example is nevertheless invalid, as this one is fixed. I will have a look at the testsuite.

**Comment by sebasguts** [comment]
@fingolfin I cannot really find a place where the testuite prints errors to the screen. Can you point this out?

**Comment by fingolfin** [comment]
@sebasguts see issue #237
